website/cue/reference/services/appsignal.cue

```diff
@@ -0,0 +1,10 @@
+package metadata
+
+services: appsignal: {
+	name:     "AppSignal"
+	thing:    "a \(name) account"
```

Comment:
I don't know what the "thing" is here. The place it is visible in AppSignal?
We have an organization and app structure (one organization has many apps), the data is eventually visible in the app-context.

Reply:
I believe the "thing" here is meant to refer to what we are sending data to, since this is a sink. Based on your description of the app-context, it does sound like changing "account" -> "app" would make sense here.